lib/src/app/navigation/app_path.dart

```diff
@@ -0,0 +1,73 @@
+abstract class AppPath {
+  const AppPath();
+}
+
+abstract class InitialPath {
+  const InitialPath();
+}
+
+abstract class BottomBarAppPath {
+  const BottomBarAppPath();
+}
+
+class HomePath extends AppPath implements InitialPath {
+  const HomePath();
+}
```

Comment:
przeniósłbym to wyżej, tak żeby było obok klasy po której dziedziczy, jak już mamy wszystko w 1 pliku to niech chociaż będzie poukładane